bindings/python/unicorn/unicorn.py

```diff
@@ -114,11 +114,11 @@ def _setup_prototype(lib, fname, restype, *argtypes):
 UC_HOOK_CODE_CB = ctypes.CFUNCTYPE(None, uc_engine, ctypes.c_uint64, ctypes.c_size_t, ctypes.c_void_p)
 UC_HOOK_MEM_INVALID_CB = ctypes.CFUNCTYPE(
     ctypes.c_bool, uc_engine, ctypes.c_int,
```

Comment:
what do you mean, @lowlyw? it is hard to follow this. if you find something wrong, can you open a new issue?